packages/cadl-ranch-specs/http/encode/union/main.tsp

```diff
@@ -0,0 +1,36 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+@doc("Test for encode decorator on union property.")
+@supportedBy("dpg")
+@scenarioService("/encode/union")
+namespace Encode.Union;
+
+model UnixTimestampDatetimeUnionProperty {
```

Comment:
I don't think this be a union property (as I don't think this is still valid if type be `utcDateTime | string`). Seems better to call it nullable property.

UnixTimestampDatetimeNullableProperty ?

Reply:
Should it just be moved into https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/encode/datetime/main.tsp ?

Reply:
agree